SSA/Projects/RISCV64/Syntax.lean

```diff
@@ -0,0 +1,676 @@
+import SSA.Core.MLIRSyntax.EDSL
+import SSA.Projects.RISCV64.Base
+
+open MLIR AST Ctxt
+open RISCV64
+
+namespace RISCVExpr
+/-!
+Defining functions to simplify Expression making for some of the RISC-V operations.
+This helps in comparing output with expected ouput and
+avoids writting huge `Expr`.
+
+-/
+def add {Γ : Ctxt _} (e₁ e₂: Ctxt.Var Γ .bv) : Expr RV64 Γ .pure .bv  :=
+  Expr.mk
+    (op := Op.add)
+    (eff_le := by constructor)
+    (ty_eq := rfl)
+    (args := .cons e₁ <| .cons e₂ .nil)
+    (regArgs := HVector.nil)
```

Comment:
I believe you don't actually use this `add` definition, or any of the other ones, in your parser.
If that's true: do we actually need them anywhere? I would imagine not, so we should remove them.

Reply:
As mentioned in the call, I would expect that just having the parser should be enough for debugging purposes (unless you're trying to debug the parser itself, of course). Still, you have more context than me, so if you believe these methods to be helpful it's totally fine to keep them.

However, if you do, please either move them to the bottom of the file (i.e., after you've finished defining the parser instances) or to a separate file. IMO the main purpose of the current file ought to be defining the parser, so I think it's nicer not to start with a lot of unrelated boilerplate.